LEAF_Request_Portal/utils/LEAF_importStandardConfig.php

```diff
@@ -5,16 +5,17 @@
 
 $debug = false;
 
-$res = $db->query_kv('SELECT * FROM settings', 'setting', 'data');
-
 // For Jira Ticket:LEAF-2471/remove-all-http-redirects-from-code
 //$protocol = isset($_SERVER['HTTPS']) ? 'https://' : 'http://';
 $protocol = 'https://';
 $siteRootURL = $protocol . HTTP_HOST;
-$relativePath = trim(str_replace($siteRootURL, '', $res['national_linkedPrimary']));
+
+$relativePath = trim(str_replace($siteRootURL, '', LEAF_SETTINGS['national_linkedPrimary']));
 $tempFolder = $_SERVER['DOCUMENT_ROOT'] . $relativePath . 'files/temp/';
+$copy_custom_templates = $_SERVER['DOCUMENT_ROOT'] . $relativePath . 'templates/email/custom_override/';
+$paste_custom_templates = getenv('APP_LIB_PATH') . '../' . PORTAL_PATH . '/templates/email/custom_override/';
 
-if($res['siteType'] != 'national_subordinate') {
+if($settings['siteType'] != 'national_subordinate') {

```

Comment:
```suggestion
if(LEAF_SETTINGS['siteType'] != 'national_subordinate') {
```

Reply:
It was not my intent to leave this for another PR. I have added them to this PR.